control-plane/api-gateway/binding/binder_test.go

```diff
@@ -54,15 +54,16 @@ var (
 )
 
 type resourceMapResources struct {
-	grants           []gwv1beta1.ReferenceGrant
-	secrets          []corev1.Secret
-	gateways         []gwv1beta1.Gateway
-	httpRoutes       []gwv1beta1.HTTPRoute
-	tcpRoutes        []gwv1alpha2.TCPRoute
-	meshServices     []v1alpha1.MeshService
-	services         []types.NamespacedName
-	consulHTTPRoutes []api.HTTPRouteConfigEntry
-	consulTCPRoutes  []api.TCPRouteConfigEntry
+	grants                   []gwv1beta1.ReferenceGrant
+	secrets                  []corev1.Secret
+	gateways                 []gwv1beta1.Gateway
+	httpRoutes               []gwv1beta1.HTTPRoute
+	tcpRoutes                []gwv1alpha2.TCPRoute
+	meshServices             []v1alpha1.MeshService
+	services                 []types.NamespacedName
+	consulInlineCertificates []api.InlineCertificateConfigEntry
```

Comment:
this was added since we were forgetting to reference count the inline certificates previously, which was leaving them orphaned in Consul when they needed to be cleaned up.